gravitee-apim-console-webui/src/management/api/general/plans/list/api-portal-plan-list.component.html

```diff
@@ -107,27 +108,57 @@
       <div class="plans__table__actions">
         <ng-container *ngIf="!isReadOnly; else readOnlyPlanActions">
           <ng-container *ngIf="element.status !== 'CLOSED'">
-            <button mat-icon-button aria-label="Edit the plan" matTooltip="Edit the plan" (click)="navigateToPlan(element.id)">
+            <button
+              mat-icon-button
+              aria-label="Edit the plan"
+              matTooltip="Edit the plan"
+              data-testid="api_plans_editplan_button"
```

Comment:
For the reference name and value I would of suggested  `data-test-id="api_plans_edit_plan_button"`
but this is probably a minor point and shouldn't block the PR

Reply:
At the beginning of the cypress tests developments the `data-testid` usage came from here https://docs.cypress.io/guides/references/best-practices#How-It-Works I think 🤷🏼 Maybe I'm wrong